src/MadLibProgram.java

```diff
@@ -0,0 +1,91 @@
+import edu.macalester.graphics.CanvasWindow;
+import edu.macalester.graphics.FontStyle;
+import edu.macalester.graphics.GraphicsText;
+import edu.macalester.graphics.ui.Button;
+
+public class MadLibProgram {
+    private MadLib currentMadLib;
+
+    private CanvasWindow canvas;
+    private final GraphicsText titleText = new GraphicsText("MADLIB GAME");
+    private final Button goButton = new Button("GO");
+    private GraphicsText madLibName = new GraphicsText();
+    private String buttonName;
+    private Button changeMadLib;
```

Comment:
These shouldn’t be instance variables. Note that they’re completely local to `addMadLibButton`.